security/azure/sql_server_va_admins/sql_server_va_admins.pt

```diff
@@ -1,13 +1,13 @@
-name "Azure Ensure SQL Server VA Notify Admins/Subscription Owners"
+name "Azure SQL Servers Vulnerability Assessment Does Not Notify Admins"
 rs_pt_ver 20180301
 type "policy"
-short_description "Report if any SQL server instances are not configured in VA to also notify admins and subscription owners. \n See the [README](https://github.com/flexera-public/policy_templates/tree/master/security/azure/sql_server_va_admins) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+short_description "Report any SQL server instances that are not configured in Vulnerability Assessment (VA) to notify admins and subscription owners. See the [README](https://github.com/flexera-public/policy_templates/tree/master/security/azure/sql_server_va_admins) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```

Comment:
```suggestion
short_description "Reports any SQL server instances that are not configured in Vulnerability Assessment (VA) to notify admins and subscription owners. See the [README](https://github.com/flexera-public/policy_templates/tree/master/security/azure/sql_server_va_admins) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```

Reply:
Not sure whether the README's description is correct or the template's one is but they read slightly differently